core/empty_response.go

```diff
@@ -1,7 +1,7 @@
 package core
 
-// DummyResponse is an empty struct. The main reason behind this
+// EmptyResponse is an empty struct. The main reason behind this
```

Comment:
If this is an empty response and it's being used to represent an empty response then let's call it an EmptyResponse. :)

Reply:
Agree, originally I used this only to see what's the real response from the server. But true, it should be Empty as we use for 204 responses too.